imdb-ratings/Controllers/GenreController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Net;
+using imdb_ratings.Models;
+using Newtonsoft.Json;
+using System;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace imdb_ratings.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GenreController : ControllerBase
+    {
+        // GET api/<GenreController>/5
+        [HttpGet("{language}")]
+        public List<string> Get(string language)
+        {
+            List<string> movieList = new List<string>();
```

Comment:
```suggestion
            List<string> movieGenreList = new List<string>();
```
you are taking genres into this not movies